physiolabxr/interfaces/DeviceInterface/TobiiProFusion/TobiiProFusion_Interface.py

```diff
@@ -0,0 +1,194 @@
+import time
+import zmq
+import subprocess
+from multiprocessing import Process, Event
+from physiolabxr.interfaces.DeviceInterface.DeviceInterface import DeviceInterface
+from physiolabxr.utils.time_utils import get_clock_time
+import numpy as np
+import os
+import platform
+
+def get_executable_path():
+    base_dir = os.path.dirname(os.path.abspath(__file__))  # Get script directory
+    if platform.system() == "Windows":
+        return os.path.join(base_dir, "x64", "Debug", "TobiiProFusion.exe")
+    else:
```

Comment:
Checking for platform should be the first line in the interface __init__(). 

This way, if the platform check fails, you won't set up Zmq and everything. This exception will be caught by the add widget, and the rest of the setup won't run. 

After you change this, please check if trying to add Tobii on mac won't cause a crash and will only give a popup window with this error message. After you close the popup window, you should be able to 
- try add tobii again and get the same response
- adding other streams (e.g., lsl dummy) without any issues